src/test/scala/chiseltest/experimental/tests/VerilatorCoverageTests.scala

```diff
@@ -23,9 +23,8 @@ class VerilatorCoverageTests extends AnyFlatSpec with ChiselScalatestTester with
     }
     val output = outputStream.toString
     coverage.exists() should be(true)
-    output.contains("--coverage-toggle") should be(true)
-    output.contains("--coverage-line") should be(false)
-    output.contains("--coverage-user") should be(false)
+    assert(output.contains("--coverage-toggle"))
```

Comment:
Can you do something like `output should contain(...)`?
(and similarly below)

Reply:
I am personally not a fan of the matchers and the original code was obviously using them wrong which meant that the error message I got from scalatest was not helpful at all. If you want me to, I can try to change it back to using matchers _correctly_, however I am not 100% certain that I know how to.

Reply:
No strong preference as long as we're somewhat locally consistent.

Reply:
I converted it to matchers.
The right matcher is `output should include(...)` since `contain` only seems to work on sequences, but not on a String.